Setup/Recurring.php

```diff
@@ -30,7 +30,7 @@ class Recurring implements InstallSchemaInterface
     /**
      * @var string
      */
-    const ORDER_STATUS_CUSTOM_LABEL = 'Pending OP Payment Service';
+    const ORDER_STATUS_CUSTOM_LABEL = 'Pending Checkout Finland';
```

Comment:
Does this handle renaming of already added custom order statuses or do we need script to rename the already existing and is there change that something brakes if there are open orders with the old status codes?

Reply:
It only sets this status for newly created orders. It should not break old codes as there is no reference to it anywhere in the code

Reply:
@richard-madarasz ok, we can do some db updating later on when we implement code changes.